spec/requests/users_spec.rb

```diff
@@ -1,27 +1,35 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    before(:example) { get users_path }
+# RSpec.describe 'Users', type: :request do
+#  describe 'GET /index' do
+#    before(:example) { get users_path }
+#
+#    it 'should be a success' do
+#      expect(response).to have_http_status(:ok)
+#    end
+#
+#    it "should render 'index' template" do
+#      expect(response).to render_template('index')
+#    end
```

Comment:
- [ ] It will be nice not to commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.